### ISSUE TITLE
[QUESTION]libpmc.so: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by s20163081161** [MIT-SPARK/TEASER-plusplus#95]
When I run ./teaser_cpp_ply, i got "error while loading shared libraries: libpmc.so: cannot open shared object file: No such file or directory"

But I can find libpmc.so in the directory "TEASER-plusplus/build/pmc-build".

Whta's wrong ?

**Comment by jingnanshi** [comment]
Try run `sudo ldconfig`

**Comment by s20163081161** [comment]
include(ExternalProject)
ExternalProject_Add(**pmc**
        GIT_REPOSITORY    https://github.com/pybind/**pybind11**.git


In the file "pybind11.CMakeLists.txt.in"，the name seems to be wrong ?

**Comment by jingnanshi** [comment]
Good catch! You can try changing `pmc` to `pybind11` and see whether it solves this issue.

**Comment by s20163081161** [comment]
@jingnanshi 
Using a pointcloud whose size is 16328, I got stuck in the step "max_clique_ = clique_solver.findMaxClique(inlier_graph_);" and finally was killed.

The algorithm do not allow big pointclouds ???

**Comment by jingnanshi** [comment]
@s20163081161 TEASAER++ takes in correspondences, not raw points. You need to use a feature detector and matcher to find correspondences.

**Comment by jingnanshi** [comment]
Doc fixed in PR #107
